realm/realm-library/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -175,11 +174,9 @@ public void querySingleRelationInteger() {
         assertEquals(1, owners6.size());
         assertEquals(12, owners6.first().getCat().getAge());
 
-        try {
-            RealmResults<Owner> owners7 = testRealm.where(Owner.class).between("cat.age", 1, 20).findAll();
-            fail();
-        } catch (IllegalArgumentException ignored) {
-        }
+        RealmResults<Owner> owners7 = testRealm.where(Owner.class).between("cat.age", 1, 20).findAll();
```

Comment:
If this is new features that we now support ranges we should track it in the CHANGELOG. Similarly on the other cases.